allennlp/service/server_sanic.py

```diff
@@ -1,23 +1,23 @@
-from allennlp.models.archival import load_archive
-from allennlp.service.predictors import Predictor
-
+from typing import Dict
 from sanic import Sanic, response, request
 from sanic.exceptions import ServerError
 
-DEFAULT_ARCHIVE_FILES = {
-        'machine-comprehension': 'tests/fixtures/bidaf/serialization/model.tar.gz',
-        'semantic-role-labeling': 'tests/fixtures/srl/serialization/model.tar.gz',
-        'textual-entailment': 'tests/fixtures/decomposable_attention/serialization/model.tar.gz'
+from allennlp.models.archival import load_archive
+from allennlp.service.predictors import Predictor
+
+DEFAULT_CONFIG = {
```

Comment:
I feel like it's weird for `DEFAULT_CONFIG` to live here but not be used in this file.

IMO, either

* `DEFAULT_CONFIG` should instead live in `serve.py`, or
* the logic for "if config not specified, then use DEFAULT_CONFIG" should be here

I don't have a strong opinion as to which of these is better